runtime_test.go

```diff
@@ -665,6 +665,49 @@ func TestRuntime_Close_ClosesCompiledModules(t *testing.T) {
 	}
 }
 
+func TestRuntime_Close_Instantiate_ShouldErr(t *testing.T) {
+	for _, tc := range []struct {
+		name            string
+		instantiateFunc func(r Runtime) error
+	}{
+		{
+			name: "InstantiateModuleFromBinary",
+			instantiateFunc: func(r Runtime) error {
+				_, err := r.InstantiateModuleFromBinary(testCtx, binaryNamedZero)
+				return err
+			},
+		},
+		{
+			name: "CompileModule",
+			instantiateFunc: func(r Runtime) error {
+				_, err := r.CompileModule(testCtx, binaryNamedZero)
+				return err
+			},
```

Comment:
redundant? @codefromthecrypt

Reply:
sorry about late feedback. I don't really think it is redundant except that the implementation of `InstantiateModuleFromBinary` happens to call this. I think it will feel less redundant when `InstantiateModule` is guarded and tested. So, if you want to swap one out I would swap out `InstantiateModuleFromBinary` for `InstantiateModule`.